android-pay/src/main/java/com/stripe/wrap/pay/utils/CartManager.java

```diff
@@ -287,14 +279,18 @@ public void setTotalPrice(@NonNull Long totalPrice) {
      */
     public void setTaxLineItem(@Nullable LineItem item) {
         if (item == null) {
+            if (mLineItemTax != null && !TextUtils.isEmpty(mLineItemTax.getTotalPrice())) {
+                setTotalPrice(null);
```

Comment:
Here and elsewhere, reflecting the fact that if you change the total price, we need to invalidate the cache.